### ISSUE TITLE
“)” is missing

### ISSUE THREAD

**Issue by fuckingdigital** [EbenSorkin/Ekta#6]
Why is there a “(” but no “)” in this font?

Thanks anyway for this charming font.

**Comment by davelab6** [comment]
Also `}`

**Comment by EbenSorkin** [comment]
I don't REALLy know. If I was guessing I would say that it was because I
made one of the glyphs use a flipped component copy of the other to save on
file size and that isn't supported in the Goggle font system anymore.

I'll update the source and push it to Git. Hopefully Google will pick it up
and make it live quickly.

Cheers!

-e.

**Comment by EbenSorkin** [comment]
I started looking at what to do to take asar into a Glyphs source file. Am
I right in assuming that if I add the missing glyphs and some OT features
that this is best done there?

-e.

**Comment by davelab6** [comment]
Yes if you want to go whole hog. If you can demo that it works offline with the file and its really the GF API that broke it that would be very interesting; my suspicion is that the TTF went wrong in export somehow.

**Comment by EbenSorkin** [comment]
I looked into it and it looks like - no. It was actually my own oversight!
It is in the source files themselves.

I plan to check if this has the same OT features and glyph names as Pria's
type on which it is based. If it does the most straightforward way of
making the Deva work properly would be to paste that OT code in. I am not
sure if it will work equally well exporting from all editors however. It
might be fine I am just not familiar with how stable OT features are across
editors.

**Comment by davelab6** [comment]
I think the immediate fix is likely some ttx surgery to get outlines into
those ttfs :)

**Comment by EbenSorkin** [comment]
If OT code can wait then I have done the fix. I'll push it.

-e.

**Comment by davelab6** [comment]
What's wrong with the OT code? :)

**Comment by EbenSorkin** [comment]
There isn't any code in the GIT sources.

I don't recall what was decided about this. If the code was meant to be
added later or if the auto code from Glyphs was the intended route.

I think https://fonts.google.com/specimen/Palanquin works and since the
forms echo those I think the glyph names probably do and so the Palanquin code
would also probably work.

But I would have to check to be sure.

-e.

**Comment by EbenSorkin** [comment]
I checked - Palanquin isn't on the Glyphs dev model it is VFB-> UFO based.

Probably updating Asar using those OT resources would be faster than
converting to Glyphs.

-e.

**Comment by davelab6** [comment]
I'm not worried about the ot code right now, assuming it basically works :)

**Comment by TimvanScherpenzeel** [comment]
Hey, thank you for making this great font!

I just noticed this issue too, did you manage to implement a fix for it? 

https://fonts.google.com/specimen/Asar still shows a missing `)` and `}`.

**Comment by EbenSorkin** [comment]
I think this was fixed in Github but I need to get google to pull it into the system.

**Comment by lahosken** [comment]
I downloaded Asar-Regular.otf from this excellent github and installed it. 

I made a little web page ; when I browse it in Chrome, it doesn't show the end-paren

```
<style>
  body {font-family: Asar; }
</style>
<body>
  Hello (World)!
</body>
```
When I use the font in Inkscape to create an SVG image, the ) does appear. But then if I use Chrome to view the SVG image, the ) does not appear.
